### PR TITLE
test(e2e): unskip safari for core components

### PR DESCRIPTION
**Pull request by maxym-valor**
## Related Issue(s)

closes

##### During Implementation

1. Visual Testing:

-   [ ] visual misalignments/updates
-   [ ] check Light/Dark/HCB/HCW themes
-   [ ] RTL/LTR - proper rendering and labeling
-   [ ] responsiveness(resize)
-   [ ] Content Density (Cozy/Compact/(Condensed))
-   [ ] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [ ] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [ ] Mouse vs. Keyboard support
-   [ ] Text Truncation

2. API and functional correctness

-   [ ] check for console logs (warnings, errors)
-   [ ] API boundary values
-   [ ] different combinations of components - free style
-   [ ] change the API values during testing

3. Documentation and Example validations

-   [ ] missing API documentation or it is not understandable
-   [ ] poor examples
-   [ ] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [ ] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [ ] tests for the changes that have been done
-   [ ] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [ ] Run npm run build-pack-library and test in external application
-   [ ] update `README.md`
-   [ ] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: e2e/wdio/core/tests/carousel.e2e-spec.ts [hunk 1]
```diff
@@ -53,11 +57,12 @@ describe('Carousel test suite', () => {
             expect(getAttributeByName(displayedImg, imgSource)).not.toBe(firstImg);
         });
 
-        it('should check page indicator dots', () => {
+        xit('should check page indicator dots', () => {
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/carousel.e2e-spec.ts:60]: does it not work for any browsers?

File: e2e/wdio/core/tests/dialog.e2e-spec.ts [hunk 2]
```diff
@@ -335,10 +341,10 @@ describe('dialog test suite', () => {
     });
 
     describe('playground examples', () => {
-        afterEach(() => {
-            refreshPage();
-            waitForElDisplayed(dialogPage.title);
-        }, 1);
+        // afterEach(() => {
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/dialog.e2e-spec.ts:344]: unskip?

File: e2e/wdio/core/tests/wizard.e2e-spec.ts [hunk 3]
```diff
@@ -205,7 +213,7 @@ describe('Wizard component test', () => {
     function checkReOpen(section: string, block: string): void {
         click(section + button);
         if (section === defaultExample) {
-            waitForElDisplayed(block + nextStep);
+            pause(5000)
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/wizard.e2e-spec.ts:216]: maybe we should keep the waitForElDisplayed here after the pause, just to make sure everything is ready for the click.

**Comment by Seamoo13** [comment]
tests failing and check-format failing, please take a look

**Comment by maxym-valor** [comment]
> tests failing and check-format failing, please take a look

format check does not work on fork at all, tests fixed

**Comment by Seamoo13** [comment]
looks like you had some issues during a rebase? 
please redo rebase to where you won't have 46 commits and changes from others included in your PR
OR just revert rebase and I'll rebase it after it's added to my changes for platform. your choice.

**Comment by maxym-valor** [comment]
> looks like you had some issues during a rebase? please redo rebase to where you won't have 46 commits and changes from others included in your PR OR just revert rebase and I'll rebase it after it's added to my changes for platform. your choice.

https://github.com/maxym-valor/fundamental-ngx/pull/117

**Pull request closed without merging**